torch/nn/attention/flex_attention.py

```diff
@@ -1132,9 +1132,9 @@ def _validate_device(query: Tensor, key: Tensor, value: Tensor):
     """TODO: Remove once non cuda/cpu devices support is added
     We only need to check query since we have already that q,k,v are on the same device
     """
-    if query.device.type != "cuda" and query.device.type != "cpu":
+    if query.device.type != "cuda" and query.device.type != "cpu" and query.device.type != "meta":
```

Comment:
This change is motivated by supporting measuring flex attention flops with meta-device inputs.